Deploying in dev of agent service to swarm fails

`./dev.sh swarm` fails when docker tries to deploy the service. Checking the service logs I see:

```
2020/05/14 09:13:59 [ERROR] [main,edge,http,cluster] [message: Unable to retrieve Edge key from cluster member] [error: Get "http://10.0.14.81:9001/key": dial tcp 10.0.14.81:9001: connect: connection refused]
portainer-agent-dev.0.20r148mi50y1@swarmnode1    | 2020/05/14 09:13:59 [ERROR] [main,edge] [message: Unable to retrieve Edge key] [error: Get "http://10.0.14.81:9001/key": dial tcp 10.0.14.81:9001: connect: connection refused]
```

The last error is printed using `log.Fatalf`, which is wrong. In this case the agent can start a UI server to get the key

--- a/cmd/agent/main.go
+++ b/cmd/agent/main.go
@@ -91,7 +91,7 @@ func main() {
 	if options.EdgeMode {
 		edgeKey, err := retrieveEdgeKey(options.EdgeKey, clusterService)
 		if err != nil {
-			log.Fatalf("[ERROR] [main,edge] [message: Unable to retrieve Edge key] [error: %s]", err)
+			log.Printf("[ERROR] [main,edge] [message: Unable to retrieve Edge key] [error: %s]", err)
 		}
 
 		if edgeKey != "" {

